### DISCUSSION TITLE
How to get provisional entries with Graphql?

### DISCUSSION THREAD

**Discussion by gopeter** [craftcms/cms#10626] [Q&A] [answered]
I'm not sure if this is a bug or if I don't get it – I hope I just don't get it 😄 

- Create an entry 
- Make some changes to that entry: 
  <img width="453" alt="image" src="https://user-images.githubusercontent.com/1242960/155151465-3f9f5791-edc6-4f98-bfcc-a054232cc6ce.png">
- Visit the preview URL: http://myapp.local/en/test?secret=mysecret&x-craft-preview=mypreview&token=5kFb9Vindh__eg8zo3jO5jOKPppwcvYs
- Look in the database: `5kFb9Vindh__eg8zo3jO5jOKPppwcvYs` has this route: `["preview/preview",{"elementType":"craft\\elements\\Entry","sourceId":5898,"siteId":1,"draftId":null,"revisionId":null,"userId":1}]`
- Now how to query the provisional content with Graphql (I'm using Craft CMS headless)? 

There is just `provisionalDrafts` but since the content change does not trigger a new draft, how to get the provisional content of a specific entry?

**Comment by brandonkelly** [comment, marked as answer]
Pass the same `token` query param along with your API requests and you should get the provisional draft data returned. No need to modify the actual query.
